### ISSUE TITLE
Dynomite cluster without using dynomite-manager

### ISSUE THREAD

**Issue by guru1306** [Netflix/dynomite#795]
Hi All

We are trying to build a highly available Dynomite+Redis Cluster (for Conductor persistence) on native Linux machines.

We want basic failure handling when Dynomite+Redis server nodes go down and come back. Does the support exist out of the box ? In some discussions, it was suggested that Dynomite-manager is mandatory for Dynomite cluster to operate, and that Dynomite-manager currently is supported only on AWS. 
 
Is it possible to run the cluster without Dynomite manager in a configuration with (DC=1, RF=3, 1 server in a rack) in a local private datacenter on Linux machines?

Thanks
Guru

**Comment by kaplingat** [comment]
+1. We too are looking for the same.

The discussion [here](https://github.com/Netflix/dynomite-manager/issues/52) seems to indicate that dynomite-manager is required for implementing a highly available Dynomite cluster (i.e. where nodes can go down and rejoin). Is this still true? The docs do not seem to be very clear on this.

**Comment by ipapapa** [comment]
Dynomite-manager is not needed to run Dynomite. Dynomite can run standalone. However, if you are thinking of a system that provides the features listed in the [readme](https://github.com/Netflix/dynomite-manager ) you will need Dynomite manager. Dynomite provides a number of Rest APIs to create your own lighweight sidecar.

**Comment by guru1306** [comment]
Thanks @ipapapa . Does this mean when a  server in a rack restarts it can catch the lost writes without Dynomite manager?

**Comment by artarik** [comment]
Nope. If you stop-start dynomite-node it lost changes which was created while node was offline. The same situation happiness if your node has network problems
